src/Services/CreateOrderService.php

```diff
@@ -331,16 +331,12 @@ public function create(ResponseDTO $dto): int
         throw new \Exception("Unable to relate payment " . $paymentID . " with order " . $orderId);
       }
 
-      // TODO: look into moving packing slip fetching to be on-demand rather than doing it during order creation.
-      // The packing slip does not need to exist in plentymarkets until the user needs it.
-      // There have been occasional timeouts in packing slip service.
-
-      $packingSlipInfo = $this->savePackingSlipService->save($orderId, $poNumber);
-
-      // TODO: investigate structure of return value from savePackingSlipService, log the ID of the created item
-
-      if (!isset($packingSlipInfo) || empty($packingSlipInfo)) {
-        throw new \Exception("Unable to add packing slip to order " . $order . " PO: " . $poNumber);
+      // packing slips are an optional feature and may be phased out.
+      // do not fail the order for lack of packing slip.
+      try {
+        $this->savePackingSlipService->save($orderId, $poNumber);
+      } catch (\Exception $exception) {
+        $externalLogs->addErrorLog("Unexpected " . get_class($exception) . " while working with packing slips: " . $exception->getMessage());
```

Comment:
Maybe log the stacktrace as well which can provide more info?

Reply:
We generally have avoided putting stacktraces into the External Logs so far. Probably just to keep it from being too noisy.

Reply:
oh right, we now have the `details` argument, so I can put the stacktrace in there!

Reply:
done.